src/ImageSearchModal.jsx

```diff
@@ -110,6 +116,15 @@ export class ImageSearchModal extends React.Component {
     render() {
         let defaultBody = (
             <React.Fragment>
+                <form className="ct-form">
```

Comment:
This needs to check whether root is actually available. If I log in without superuser, I still see these radio buttons. The search then spins forever

Reply:
I am aware of this, it is in the `followups list`. But I may implement it here as well

Reply:
Fixed.